app/routes/task_routes.py

```diff
@@ -0,0 +1,107 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.task import Task
+from datetime import date
+import os
+import requests
+from .helper_function import get_one_obj_or_abort
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+#--------------------------------helper functions---------------------------
+
+def send_to_slack(task):
+    url = "https://slack.com/api/chat.postMessage?channel=task-notifications&text=%22Hello%20World%22&pretty=1"
+    params = {
+        "channel": "task-notifications",
+        "text": f"Someone just completed the task {task.title}"
+    }
+    header = {"Authorization": os.environ.get("SLACK_API_TOKEN")}
+
+    requests.post(url, data= params, headers= header)
+
+#---------------------------POST-------------------------------------------- 
+@task_bp.route("", methods=["POST"])
+def add_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or\
+        "description" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
```

Comment:
This validation would be great as a helper function and also indicate which data is invalid (title or description).   Providing a meaningful error message can be important.